members/C001113.yaml

```diff
@@ -55,12 +55,8 @@ contact_form:
           selector: textarea#input-78DF3A4F-C8ED-2264-6E6C-A51B1269E6D2
           value: $MESSAGE
           required: true
-    - javascript:
-        - name: char replace
-          values: ["$MESSAGE"]
-          selectors: ["textarea#input-78DF3A4F-C8ED-2264-6E6C-A51B1269E6D2"]
-          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '');}"]
-          required: true
+          options:
+            blacklist: "-,;—:()"
```

Comment:
i think you still need this list to prevent the characters. does this form work with any of these characters now?